apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -42,6 +42,9 @@ const (
 	ClusterDriverImported = "imported"
 	ClusterDriverLocal    = "local"
 	ClusterDriverRKE      = "rancherKubernetesEngine"
+	ClusterDriverGKE      = "googleKubernetesEngine"
```

Comment:
looks like this PR is based off cluster capabilities changes; can we make based off master for now so the changes are clear?